src/store/getters/state-getters.ts

```diff
@@ -793,26 +807,6 @@ export const havePeopleAndRolesChanged = (state: StateIF): boolean => {
   return !IsSame(currentOrgPersons, originalOrgPersons, ['actions', 'confirmNameChange'])
 }
 
-/** True when the minimum proprietors is met. */
-export const hasMinimumProprietor = (state: StateIF): boolean => {
-  // REMOVED parties are still in the parties array until FILING, so exclude them for component level
-  // validations
-  return (
-    !isEntityTypeSP(state) ||
-    getOrgPeople(state).filter(party => !party.actions?.includes(ActionTypes.REMOVED)).length === 1
-  )
-}
-
-/** True when the minimum partners met. */
-export const hasMinimumPartners = (state: StateIF): boolean => {
-  // REMOVED parties are still in the parties array until FILING, so exclude them for component level
-  // validations
-  return (
-    !isEntityTypeGP(state) ||
-    getOrgPeople(state).filter(party => !party.actions?.includes(ActionTypes.REMOVED)).length >= 2
-  )
-}
```

Comment:
As per comment above, these were moved to PeopleAndRoles.vue where they're used.

Actually, I think we should keep THIS file to store getters only, and have a separate file (mixin?) for more complicated global getter logic. Questions? Thoughts?

Reply:
I agree, it would make transitioning to the Composition API easier when that needs to be done.